.github/workflows/ci.yaml

```diff
@@ -11,28 +11,17 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Setup PATH
-        shell: bash
-        run: |
-          mkdir -p "$HOME/.local/bin"
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
+      - uses: actions/checkout@v4
 
       - name: Install jsonnet
-        shell: bash
```

Comment:
The addition of `shell: bash` is relevant for github action definitions, but not in a github workflow file - the default is to use bash.